### DISCUSSION TITLE
How to avoid swap file warning E325?

### DISCUSSION THREAD

**Discussion by eddyecho** [willothy/flatten.nvim#82] [Q&A]
If I already have a file open in an existing buffer and then try to open the file again in terminal mode with simple `nvim file`, how do i avoid an E325? I am coming from neovim remote which handled this out of box so I'm wondering if I am missing something here?

**Comment by willothy** [comment]
I am not sure, but I will look into how nvim-remote handles this! I believe there's a patch in nvim v10 that fixes this issue by ignoring swapfiles for files open in active nvim instances, but it would be nice to have that functionality for older nvim versions.
